### PR TITLE
Initial Docker commit

### PR DESCRIPTION
**Pull request by christronyxyocum**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	php7-sqlite3 \
+	php7-session \
+	php7-zip
+
+# Add local files
+COPY root/ /
+
+# Port and volumes
+EXPOSE 80
+VOLUME /config
```
Comment (si0972) [inline Dockerfile]: Expose `/var/log` too?

File: root/defaults/default [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	root /config/www/Monitorr;
+	index index.html index.htm index.php;
+
+	server_name _;
+	client_max_body_size 0;
+
+	location / {
+		try_files $uri $uri/ /index.html /index.php?$args =404;
+	}
+	location ~ \.php$ {
```
Comment (si0972) [inline root/defaults/default]: Consider pump up php-children?

File: root/defaults/default [hunk 3]
```diff
@@ -0,0 +1,22 @@
+server {
+	listen 80 default_server;
+	root /config/www/Monitorr;
```
Comment (si0972) [inline root/defaults/default]: why be inconsistent with capital letters?

File: Dockerfile (hunk 1 above)
Comment (christronyxyocum) [reply Dockerfile]: I don't think it has to be exposed like /config. With these base images, you need to map /var/logs or they won't start. I just need to update the README and Docker hub page to reflect that.

**Comment by seanvree** [comment]
@christronyxyocum can you re-do this PR to this branch:  https://github.com/Monitorr/Monitorr/tree/docker

**Pull request closed without merging**